src/hevm/src/EVM/ABI.hs

```diff
@@ -24,93 +25,88 @@
-  , abiValueType
-  , abiTypeSolidity
-  , abiMethod
-  , emptyAbi
-  , encodeAbiValue
-  , decodeAbiValue
-  , decodeStaticArgs
-  , decodeBuffer
-  , formatString
-  , parseTypeName
-  , makeAbiValue
-  , parseAbiValue
-  , selector
-  ) where
-
-import EVM.Types
-
-import Control.Monad      (replicateM, replicateM_, forM_, void)
-import Data.Binary.Get    (Get, runGet, runGetOrFail, label, getWord8, getWord32be, skip)
-import Data.Binary.Put    (Put, runPut, putWord8, putWord32be)
-import Data.Bits          (shiftL, shiftR, (.&.))
-import Data.ByteString    (ByteString)
-import Data.DoubleWord    (Word256, Int256, signedWord)
-import Data.Functor       (($>))
-import Data.Text          (Text, pack, unpack)
-import Data.Text.Encoding (encodeUtf8, decodeUtf8')
-import Data.Vector        (Vector, toList)
-import Data.Word          (Word32)
-import Data.List          (intercalate)
-import Data.SBV           (SWord, fromBytes)
-import GHC.Generics
+  ( AbiValue (..),
+    AbiType (..),
+    AbiKind (..),
+    AbiVals (..),
+    abiKind,
+    Event (..),
+    Anonymity (..),
+    Indexed (..),
+    putAbi,
+    getAbi,
+    getAbiSeq,
+    genAbiValue,
+    abiValueType,
+    abiTypeSolidity,
+    abiMethod,
+    emptyAbi,
+    encodeAbiValue,
+    decodeAbiValue,
+    decodeStaticArgs,
+    decodeBuffer,
+    formatString,
+    parseTypeName,
+    makeAbiValue,
+    parseAbiValue,
+    selector,
+  )
+where
 
-import Test.QuickCheck hiding ((.&.), label)
-import Text.ParserCombinators.ReadP
 import Control.Applicative
-
-import qualified Data.ByteString       as BS
+import Control.Monad (forM_, replicateM, replicateM_, void)
+import Data.Binary.Get (Get, getWord32be, getWord8, label, runGet, runGetOrFail, skip)
+import Data.Binary.Put (Put, putWord32be, putWord8, runPut)
+import Data.Bits (shiftL, shiftR, (.&.))
+import Data.ByteString (ByteString)
+import qualified Data.ByteString as BS
 import qualified Data.ByteString.Char8 as Char8
-import qualified Data.ByteString.Lazy  as BSLazy
-import qualified Data.Text             as Text
-import qualified Data.Vector           as Vector
-
-import qualified Text.Megaparsec      as P
+import qualified Data.ByteString.Lazy as BSLazy
+import Data.DoubleWord (Int256, Word256, signedWord)
+import Data.Functor (($>))
+import Data.List (intercalate)
+import Data.SBV (fromBytes)
+import Data.Text (Text, pack, unpack)
+import qualified Data.Text as Text
+import Data.Text.Encoding (decodeUtf8', encodeUtf8)
+import Data.Vector (Vector, toList)
+import qualified Data.Vector as Vector
+import Data.Word (Word32)
+import EVM.Types
+import GHC.Generics
+import Test.QuickCheck hiding (label, (.&.))
+import qualified Text.Megaparsec as P
 import qualified Text.Megaparsec.Char as P
+import Text.ParserCombinators.ReadP
 
 data AbiValue
-  = AbiUInt         Int Word256
-  | AbiInt          Int Int256
-  | AbiAddress      Addr
-  | AbiBool         Bool
-  | AbiBytes        Int BS.ByteString
+  = AbiUInt Int Word256
```

Comment:
I don't think we should let the linter dictate code alignment

Reply:
undone 👯